### ISSUE TITLE
Console Mode Read Password

### ISSUE THREAD

**Issue by hypersoft** [hypersoft/Krypton#9]
There is no support for this operation.

**Comment by hypersoft** [comment]
Support for this method is not available due to the way Java implements the feature.

1. Obtaining the console object is not guaranteed to succeed.
2. The readPassword method of the console object doesn't delete the last line entered and this causes ugly formatting issues with krypton's read-eval-print-loop within the context of line mode operations, because there is no way for krypton to detect the last line, and there is no known way for XPR to scrub the last line post operation completion.

basically, java's standard implementation of console does not meet this project's requirements. Support for this is a low-priority. The recommended course of action is to use XPR.run, targeting a system command/utility which provides the CORRECT functionality.
